app/src/main-process/menu/build-default-menu.ts

```diff
@@ -238,16 +238,40 @@ export function buildDefaultMenu({
         accelerator: 'CmdOrCtrl+0',
         click: zoom(ZoomDirection.Reset),
       },
+      {
+        label: __DARWIN__ ? 'Reset Zoom' : 'Reset zoom',
+        accelerator: 'CmdOrCtrl+num0',
+        click: zoom(ZoomDirection.Reset),
+        visible: false,
+      },
+      {
+        label: __DARWIN__ ? 'Zoom In' : 'Zoom in',
+        accelerator: 'CmdOrCtrl+Plus',
+        click: zoom(ZoomDirection.In),
+      },
       {
         label: __DARWIN__ ? 'Zoom In' : 'Zoom in',
         accelerator: 'CmdOrCtrl+=',
         click: zoom(ZoomDirection.In),
+        visible: false,
```

Comment:
```suggestion
        label: __DARWIN__ ? 'Zoom In' : 'Zoom in',
        accelerator: 'CmdOrCtrl+Plus',
        click: zoom(ZoomDirection.In),
        visible: false,
      },
{
label: __DARWIN__ ? 'Zoom In' : 'Zoom in',
accelerator: 'CmdOrCtrl+=',
click: zoom(ZoomDirection.In),
```

I understand this is inline with the issue.. and.. since not all users will have a number pad, we should prefer to show the keyboard shortcut that is available across all keyboards with is `CmdOrCtrl+=`